website/views/layouts/layout.ejs

```diff
@@ -283,29 +283,40 @@
     <%/* Stripe.js */%>
     <script src="https://js.stripe.com/v3/"></script>
 
-    <%/* Linkedin Ads Insight tag */%>
-    <script type="text/javascript">
-    _linkedin_partner_id = "4365817";
-    window._linkedin_data_partner_ids = window._linkedin_data_partner_ids || [];
-    window._linkedin_data_partner_ids.push(_linkedin_partner_id);
-    </script><script type="text/javascript">
-    (function(l) {
-    if (!l){window.lintrk = function(a,b){window.lintrk.q.push([a,b])};
-    window.lintrk.q=[]}
-    var s = document.getElementsByTagName("script")[0];
-    var b = document.createElement("script");
-    b.type = "text/javascript";b.async = true;
-    b.src = "https://snap.licdn.com/li.lms-analytics/insight.min.js";
-    s.parentNode.insertBefore(b, s);})(window.lintrk);
-    </script>
-    <noscript>
-    <img height="1" width="1" style="display:none;" alt="" src="https://px.ads.linkedin.com/collect/?pid=4365817&fmt=gif" />
-    </noscript>
-
     <% /* Delete the global `self` to help avoid client-side bugs.
     (see https://developer.mozilla.org/en-US/docs/Web/API/Window/self) */ %>
     <script>delete window.self;</script>
 
+    <% /* Sticky header */ %>
+    <script>
+        var lastScrollTop = 0;
+        var header = document.querySelector('.header') || document.querySelector('.homepage-header')
+
+        window.addEventListener('scroll', windowedScrolled)
+
+        function windowedScrolled() {
+            var scrollTop = window.pageYOffset || document.documentElement.scrollTop
+            var fleetLogo = header.querySelector('img')
+            if(scrollTop == 0 && header.classList.contains('homepage-header')) {
+                header.classList.remove('sticky')
+                fleetLogo.src = '/images/logo-white-118x48@2x.png'
+                fleetLogo.style = 'width: 118px; height: 48px;'
+                return;
+            }
+            if (scrollTop > lastScrollTop && scrollTop > window.innerHeight*1.5) {
```

Comment:
I don't know if this is the correct syntax, but I want the masthead to stay in position for more than a window height.

Reply:
Oh, that makes sense. I went for that as well previously but just thought the height of a window worked.

Reply:
And you are doing great as a programmer 😉